pkg/composer/down.go

```diff
@@ -96,7 +96,18 @@ func (c *Composer) downNetwork(ctx context.Context, shortName string) error {
 	netExists, err := c.NetworkExists(fullName)
 	if err != nil {
 		return err
-	} else if netExists {
+	}
+
+	if netExists {
+		netUsed, err := c.NetworkUsed(ctx, fullName)
+		if err != nil {
+			return err
+		}
+
+		if netUsed {
+			return fmt.Errorf("failed to remove network %q: Error response from daemon: error while removing network: network %q has active endpoints", fullName, fullName)
```

Comment:
The error message here could be the same with https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/network_rm.go#L76.

Just make the message as simple as we can

Reply:
make sense

> The error message here could be the same with https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/network_rm.go#L76.
> 
> Just make the message as simple as we can